lib/github.js

```diff
@@ -99,7 +99,14 @@ class GithubRetriever {
     }
 
     this.log.info('Getting repos for %s', org)
-    const repos = await this.got.paginate.all(`orgs/${org}/repos`, { token })
+    // It's possible at some point in the future, github api https://developer.github.com/v3/repos/
+    // will allow us to filter for non archived repos during the request. Until then, we'll fetch all
+    // repos and filter the response.
+    let repos = await this.got.paginate.all(`orgs/${org}/repos`, { token })
```

Comment:
nit: you can actually filter inside the paginate options (see got docs). we are doing something similar in another call in this file. the benefit would be that we aren't putting the entire repo list into memory before filtering the archived ones out.